client/src/Plant/PlantForm.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+
+import Api from "../Api";
+
+function PlantForm(){
+    const {id} = useParams();
+    const history = useHistory();
+    const [plant, setPlant] = useState({
+        name: '',
+        slug: '',
```

Comment:
slug and position were copied from sections/skills in full-stack-starter, but are not the attributes/columns being edited in the Plant model.